gadgets/src/integers/uint/tests/uint128.rs

```diff
@@ -454,22 +440,4 @@ fn test_uint128_pow() {
     assert!(r.value == Some(expected));
 
     check_all_allocated_bits(expected, r);
-
-    // Flip a bit_gadget and see if the exponentiation constraint still works
-    if cs
-        .get("exponentiation/multiply_by_self_0/partial_products/result bit_gadget 0/boolean")
-        .is_zero()
-    {
-        cs.set(
-            "exponentiation/multiply_by_self_0/partial_products/result bit_gadget 0/boolean",
-            Fr::one(),
-        );
-    } else {
-        cs.set(
-            "exponentiation/multiply_by_self_0/partial_products/result bit_gadget 0/boolean",
-            Fr::zero(),
-        );
-    }
-
-    assert!(!cs.is_satisfied());
```

Comment:
Ditto - if there is a chance we'll use `TestConstraintSystem` for `i128` in the future, I'd just comment this out instead. Otherwise, it can be removed.

Reply:
Good points. Let me comment them out and leave a note on why they are temporarily hidden.